app/assets/stylesheets/styles/components/guides/_new_guides.css.scss

```diff
@@ -103,6 +103,30 @@
     margin: 1.5rem 0;
   }
 
+  .life-stage-nav {
+    background-color: #54853f;
```

Comment:
Color literals like `#54853f` should only be used in variable declarations; they should be referred to via variable everywhere else.